### ISSUE TITLE
Inventory Duplication Bug

### ISSUE THREAD

**Issue by Kazzababe** [Kazzababe/TekkitInspired#11]
"This message is important (without laughing)!

Players on my server found a duplicate bug with the quarry! They can duplicate any item, and this completely destroyed the economy on my server. To exploit this bug the players must be two. One of the two players places the item he wants to duplicate in the inventory of the quarry, then he keeps the inventory open. Meanwhile the other player breaks the quarry which then loots its contents (the duplicated item). The bug is that the inventory does not close for the other player who can retrieve what he has placed.

To patch this one must find when a quarry breaks, and at this time close its inventory for the players.

PS: This probably works with other machines that have an inventory."

**Comment by Th0rgal** [comment]
Hey, I had this bug a few years ago when I was using your plugin. Are you still working on it? If you are not, are looking for someone to continue the project?

**Comment by EverNife** [comment]
last commit was from 4 years ago....
Seems pretty "uncontinued" to me :D

**Comment by Kazzababe** [comment]
Yep, it never really picked off and I had other things to do, and I lost motivation to continue working on it.

**Comment by Kazzababe** [comment]
I don't know if anyone is actually still interested in this project, but I actually did make a new version last night: https://imgur.com/a/SJoXbFm

**Comment by Th0rgal** [comment]
Damn that's cool, what did you add? :o

**Comment by Kazzababe** [comment]
Pipes now show the items and are done via armor stand packets. I'd re-implement all the old machines, clean up the recipes, etc, etc. Add some new ones and probably add some custom textures to the blocks and I'm not sure what else.

**Comment by Th0rgal** [comment]
Wow it would be awesome. Will you fix the dupe bug? 
Btw I'm the developers of an opensource plugin called Oraxen. You might be interested in a compatibility with yours. It allows to add new items and blocks with custom textures easily (it generates a resourcepack automatically from the config). It would be nice to have those machines with custom textures 😁
